csharp/Ice/hello/Hello.ice

```diff
@@ -8,7 +8,7 @@ module Demo
 {
     interface Hello
     {
-        idempotent void sayHello(int delay);
-        void shutdown();
+        void sayHello();
+        string sayHi();
```

Comment:
I think these method names are confusing. Maybe `sayHello` and `receiveHello`? Maybe we could have a single operation that takes a `struct Person` and returns a hello message `"Hello, FirstName LastName"`?

Reply:
we use `idempotent string getGreeting();` in `IceDiscovery\replication\Hello.ice`

Reply:
I think the method names could use improvement, but I think it's best to leave off `idempotent` here. Technically it's correct to mark it idempotent, but this is supposed to be a bare-bones demo for people who are completely new to ice.

Adding idempotent (which is not a normal keyword in any other language I know of) adds complexity here and isn't a useful concept to the user when they're just starting out with ice I think.